src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -1,78 +1,87 @@
+          </mat-card>
+
+          <div
+            *ngIf="topicOption.type !== TopicSelectionEnum.MANUAL || isGitHubAppEntry; else editableManualField"
+            [class]="topicOption.value ? '' : 'gray-caption'"
           >
-            <input
-              matInput
-              [(ngModel)]="editedTopicManual"
-              [disabled]="isGitHubAppEntry"
-              placeholder="Short description of the {{ entryTypeMetadata.term }}"
-              data-cy="topicInput"
-            />
-          </mat-form-field>
-        </mat-radio-button>
-      </mat-card>
-      <mat-card *ngIf="!isHostedEntry">
-        <mat-radio-button [value]="TopicSelectionEnum.AUTOMATIC" class="w-100">
-          <div>
-            <span class="bubble mr-2"><mat-icon class="mat-icon-sm">sync</mat-icon></span>
-            <strong>Automatic</strong>
+            {{ topicOption.value || 'Not Available' }}
           </div>
-          <div class="size-small gray-caption">Retrieved automatically from the GitHub repository description.</div>
-          <div [class]="entry.topicAutomatic ? '' : 'gray-caption'">
-            {{ entry.topicAutomatic || 'Not Available' }}
-          </div>
-        </mat-radio-button>
-      </mat-card>
-      <mat-card>
-        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button">
-          <div>
-            <span class="bubble mr-2"><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon" /></span>
-            <strong>AI</strong>
-          </div>
-          <div class="size-small gray-caption">Generated by AI using the content of your {{ entryTypeMetadata.term }}.</div>
-          <div [class]="entry.topicAI ? '' : 'gray-caption'">{{ entry.topicAI || 'Not Available' }}</div>
+          <ng-template #editableManualField>
+            <mat-form-field
+              color="accent"
+              appearance="outline"
+              class="w-100"
+              matTooltip="Edit the manual topic for this {{ entryTypeMetadata.term }}"
+            >
+              <input
+                matInput
+                [(ngModel)]="editedTopicManual"
+                placeholder="Short description of the {{ entryTypeMetadata.term }}"
+                data-cy="topicInput"
+              />
+            </mat-form-field>
+          </ng-template>
         </mat-radio-button>
       </mat-card>
     </mat-radio-group>
   </div>
 </mat-dialog-content>
-<div mat-dialog-actions class="pull-right">
+<div mat-dialog-actions fxLayout="row" fxLayoutAlign="end">
+  <div *ngIf="promptToConfirmAISelection" fxLayout="row" fxLayoutAlign="end" class="mb-3">
+    <mat-card class="alert-warning mat-elevation-z size-small weight-bold py-1" fxFlex="50"
+      >By selecting the AI topic, you are confirming that you have reviewed and approved it. Are you sure you want to select the AI
```

Comment:
I wonder if we should explain that, after confirming, the "ai generated" bubble will no longer be displayed to the public.